JSTests/wasm/gc/simd.js

```diff
@@ -1,4 +1,5 @@
 //@ skip unless $isSIMDPlatform
+//@ requireOptions("--useWasmSIMD=1")
```

Comment:
Is this right? Shouldn't these force `--useWasmJIT=1` until we have IPInt working?

Reply:
I think Daniel is using `--useWasmSIMD=1` as an annotation (as a hack) to exclude these tests (with some pre-processing).  I think it would be wrong to require `--useWasmJIT=1` because it would communicate that this test must be run with the JIT, which eventually, it won't be.

However, it we don't need `--useWasmJIT=1` for anything else, then it can be the hack annotation, and we can don't need a preprocessor to filter these out.

Reply:
I see... I'm not sure that's any less confusing or more likely to ensure we remove the "skip". IMO, we should just add a `def annotateSIMDTest` to `run-jsc-stress-tests` that disables the test in `no-wasm-jit` config. It should probably have a comment to remove `annotateSIMDTest` once we enable IPInt SIMD, as `annotateSIMDTest` stands out and is easy to grep for.

Reply:
The advantage of `--useWasmSIMD=1` is that many relevant tests are already annotated so.  And it does correct ly document the test's expectation.  I think this is an acceptable hack for the short term.

Reply:
Yeah, I don't feel too strongly. Although, I think we should file a radar to remove the hack and block it on IPInt.